scripts/apps/workspace/content/components/ContentProfileFieldsConfig.tsx

```diff
@@ -0,0 +1,634 @@
+
+                    setIndexForNewItem(newItemIndex);
+                    page.openNewItemForm({_id: fieldId});
+                }}
+                style={{width: 100}}
+            >
+                <option value="">{gettext('Add field')}</option>
+                {
+                    availableIds.map(({id, label}) => (
+                        <option key={id} value={id}>{label}</option>
+                    ))
+                }
+            </select>
+        );
+
+        return (
+            <div
+                style={{
+                    position: 'relative',
+
+                    // a clone of the element gets appended to the body when sorting is in progress
+                    // in order to be visible, it has to have a higher zIndex than the modal
+                    zIndex: 1051,
+
+                    border: `1px solid ${inEditMode ? 'red' : 'blue'}`,
+                    marginTop: 10,
+                    marginBottom: 10,
+                    paddingTop: 5,
+                    paddingBottom: 5,
+                    cursor: 'pointer',
+                    userSelect: 'none',
+                }}
+                onClick={(e: any) => {
+                    if (
+                        querySelectorParent(e.target, 'select', {self: true}) == null
+                        && querySelectorParent(e.target, 'button', {self: true}) == null
+                    ) {
+                        page.startEditing(item._id);
+                    }
+                }}
+            >
+                {
+                    !sortingInProgress
+                        ? (
+                            <div style={{
+                                display: 'flex',
+                                justifyContent: 'center',
+                                width: '100%',
+                                position: 'absolute',
+                                top: '-19px',
+                            }}>
+                                {newFieldSelect(index)}
+                            </div>
+                        )
+                        : null
+                }
+
+                {getLabel(item.id)}
+                <button onClick={() => page.deleteItem(item)}>{gettext('Delete')}</button>
+
+                {
+                    item.required === true ? (<strong>{gettext('required')}</strong>) : null
+                }
+
+                {
+                    !sortingInProgress && isLast
+                        ? (
+                            <div style={{
+                                display: 'flex',
+                                justifyContent: 'center',
+                                width: '100%',
+                                position: 'absolute',
+                                bottom: '-17px',
+                            }}>
+                                {newFieldSelect(index + 1)}
+                            </div>
+                        )
+                        : null
```

Comment:
ok I see you use the ternary operator a lot, is there any reason for that? do you think it's more readable or that it's easier to add things in the future?